molecule/shared/cleanup.yml

```diff
@@ -0,0 +1,8 @@
+- name: unregister systems
+  hosts: all
+  roles:
+    - role: oasis_roles.rhsm
+      when: ansible_distribution == 'RedHat'
+  vars:
+    rhsm_unregister: true
+    rhsm_username: "{{ omit }}"
```

Comment:
It's probably safe to preemptively omit this omit, and I'll get https://github.com/oasis-roles/rhsm/issues/22 taken care of shortly.

Reply:
I'm thinking it might be best to default to omitting this particular file from the change as it requires dependency to be run before cleanup which means earlier in test than it currently runs. This might be another thing that's waiting on the next Molecule release.

Reply:
OK, I've cleaned this up as well as including the better run logic you mentioned.